### PR TITLE
Add option to make links lowercase

### PR DESCRIPTION
**Pull request by DatGuy1**
Seems like clang autoformatted some stuff

### CODE REVIEW

**Comment by fourtf** [comment]
The setting is supposed to only lowercase the domain name and not the entire url

**Comment by fourtf** [comment]
Looks good but I haven't tested it yet

File: src/providers/twitch/TwitchMessageBuilder.cpp [hunk 1]
```diff
@@ -235,6 +235,19 @@ MessagePtr TwitchMessageBuilder::build()
                 if (linkString.isEmpty()) {
                     link = Link();
                 } else {
+                    if (app->settings->lowercaseLink) {
+                        QRegularExpression httpRegex("\\bhttps?://");
+                        QRegularExpression ftpRegex("\\bftps?://");
```
Comment (pajlada) [inline src/providers/twitch/TwitchMessageBuilder.cpp]: These two regular expressions need `QRegularExpression::CaseInsensitiveOption` options added

**Comment by pajlada** [comment]
I will yolo merge, since I think it will work combined with the upstream changes to link parsing

**Pull request merged**